onnxruntime/test/providers/cpu/reduction/reduction_ops_test.cc

```diff
@@ -35,6 +35,9 @@ void TestReduceOp(const std::string& op,
   test.Run(OpTester::ExpectResult::kExpectSuccess, "", {kCudaExecutionProvider, kTensorrtExecutionProvider});  //TensorRT: result differs
 }
 
+//TODO:investigate why it is so slow. It need 12 seconds on an Azure Standard F48s_v2 (48 vcpus, 96 GiB memory)
+// machine in RelWithDebInfo build mode, but only 2 seconds on my local dev machine(4 cores).
+#ifdef NDEBUG
 TEST(ReductionOpTest, ReductionVariationTest) {
```

Comment:
I don't think we should disable this as doing so will mean it gets ignored and we never investigate why it's slow.